### ISSUE TITLE
Donation CRUD

### ISSUE THREAD

**Issue by angelina-7** [podkrepi-bg/api#100]
Implement CRUD operations for Donation entity.

**Comment by PepiPetrov** [comment]
Hello, @Polina1985! I am not working with @angelina-7, but if there is something I can help with, I'll try to answer.

**Comment by Polina1985** [comment]
Could you please explain the idea and the info of Menu "Дарения" in the Admin panel.

Should the info be added automatically after the DONATION THROUGH the platform is made?
I THINK it is relevant to have the following info:

1. Donator - Name/ Company name
2. Sum - currency
3. Bank 
4. Date and hour of the transaction


PROPOSAL:
- NO to have "delete" option
- "export" function - as excel
- "print" option
- sort function / search function - for ex. search by "name of Donator" - Polina

**Comment by angelina-7** [comment]
Yes, @Polina1985, the info is all donations made through the platform.
I'll make the grid display the suggested fields. As for the 'delete' option, I consulted with the team and decided to change the status only without removing the record.
Export and print options will be added too, I'll make new issues for them.
Sorting and filtering are part of the grid itself. I've attached a screenshot.
![Screenshot 2022-03-24 132217](https://user-images.githubusercontent.com/62725741/159936922-49c2a47a-3a87-4431-ba08-57e87d93ae76.jpg)
